### PR TITLE
Document the model outputs and easier installation instructions

### PR DESCRIPTION
**Pull request by giovannic**

### CODE REVIEW

File: R/model.R [hunk 1]
```diff
@@ -1,15 +1,54 @@
 #' @title Run the simulation
 #' @description
-#' The main entrypoint for the simulation. run_simulation puts together the
-#' model components and runs the malaria simulation. This currently returns a
```
Comment (htopazian) [inline R/model.R:4]: I like the line "This currently returns a dataframe with the number of individuals in each state at each timestep."

File: R/model.R [hunk 2]
```diff
@@ -1,15 +1,54 @@
...
+#' Run the simulation for some time given some parameters.
 #'
-#' Warning: the columns of the output dataframe is likely to change as we figure
-#' out what kind of outputs we would like to report from the simulation.
+#' The resulting dataframe contains the following columns:
 #'
-#' @param timesteps the number of timesteps to run the simulation for
+#' timestep: the timestep for the row
+#' infectivity: the infectivity from humans towards mosquitoes
+#' lambda: the effective biting rate on humans (per timestep) (per
```
Comment (htopazian) [inline R/model.R:9]: indicates that it defaults to "lambda_All"

File: R/model.R [hunk 3]
```diff
@@ -1,15 +1,54 @@
...
+#' species)
+#' normal_lambda: the effective biting rate on adult humans (per
```
Comment (htopazian) [inline R/model.R:11]: indicate that it defaults to "normal_lambda_All"

File: R/model.R [hunk 4]
```diff
@@ -1,15 +1,54 @@
...
+#' S_count: number of humans who are Susceptible
+#' A_count: number of humans who are Asymptomatic
+#' D_count: number of humans who have the clinical malaria
+#' U_count: number of subpatent infections in humans
+#' Tr_count: number of infections being treated in humans
+#' ica_mean: the mean acquired immunity to clinical infection over the population of humans
+#' icm_mean: the mean maternal immunity to clinical infection over the population of humans
+#' ib_mean: the mean blood immunity to all infection over the population of humans
+#' id_mean: the mean immunity from detection over the population of humans
+#' pv: prevalence for humans between an inclusive age range (in timesteps)
```
Comment (htopazian) [inline R/model.R:29]: Indicate that it defaults to "pv_730_3650" but the variable name is dependent on the values set for prevalence_rendering_max_ages and prevalence_rendering_min_ages

File: R/model.R [hunk 5]
```diff
@@ -1,15 +1,54 @@
...
+#' inc: incidence for humans between an inclusive age range (in timesteps)
```
Comment (htopazian) [inline R/model.R:30]: indicate that inc only appears in the dataset if incidence_rendering_min_ages, incidence_rendering_max_ages are set. 

and the same for sev_inc - that severe_enabled is set to 1 and severe_incidence_rendering_max_ages and severe_incidence_rendering_min_ages have values. 

but if we want to include sev_inc, we also should include sev_deaths when severe_enabled is on and pv_severe when severe_prevalence_rendering_min_ages and severe_prevalence_rendering_max_ages have values.

File: R/model.R [hunk 6]
```diff
@@ -1,15 +1,54 @@
...
+#' clin_inc: clinical incidence for humans between an inclusive age range (in timesteps)
+#' sev_inc: severe incidence for humans between an inclusive age range (in timesteps)
+#' E_count: number of mosquitoes in the early larval stage
+#' L_count: number of mosquitoes in the late larval stage
+#' P_count: number of mosquitoes in the pupal stage
+#' Sm_count: number of adult female mosquitoes who are Susceptible
+#' Pm_count: number of adult female mosquitoes who are incubating
+#' Im_count: number of adult female mosquitoes who are infectious
+#' total_M: number of adult female mosquitoes
```
Comment (htopazian) [inline R/model.R:39]: also total_M_1 - is that for the species number (defaulting to "All")?

File: README.md [hunk 7]
```diff
@@ -14,7 +14,21 @@ the [individual](https://github.com/mrc-ide/individual) package.
 
 Please note, malariasimulation is only compatible with R >= 4.0.0
 
-The package can be installed from github using the "remotes" library
+You can install the binary package on Windows and Mac using the following
+command:
+
+```R
+# Enable this universe
+options(repos = c(
```
Comment (htopazian) [inline README.md:22]: Giovanni, can you add a comment to explain why you need to enable the universe? Is this just an alternative option to using remotes or is it necessary?

File: R/model.R [hunk 8]
```diff
@@ -1,15 +1,54 @@
...
+#' D_count: number of humans who have the clinical malaria
+#' U_count: number of subpatent infections in humans
+#' Tr_count: number of infections being treated in humans
+#' ica_mean: the mean acquired immunity to clinical infection over the population of humans
+#' icm_mean: the mean maternal immunity to clinical infection over the population of humans
+#' ib_mean: the mean blood immunity to all infection over the population of humans
+#' id_mean: the mean immunity from detection over the population of humans
+#' pv: prevalence for humans between an inclusive age range (in timesteps)
+#' inc: incidence for humans between an inclusive age range (in timesteps)
+#' clin_inc: clinical incidence for humans between an inclusive age range (in timesteps)
```
Comment (htopazian) [inline R/model.R:31]: I don't see this one in my output

File: R/model.R [hunk 9]
```diff
@@ -1,15 +1,54 @@
...
-#' dataframe with the number of individuals in each state at each timestep
+#' Run the simulation for some time given some parameters.
 #'
-#' Warning: the columns of the output dataframe is likely to change as we figure
-#' out what kind of outputs we would like to report from the simulation.
+#' The resulting dataframe contains the following columns:
 #'
-#' @param timesteps the number of timesteps to run the simulation for
+#' timestep: the timestep for the row
+#' infectivity: the infectivity from humans towards mosquitoes
```
Comment (htopazian) [inline R/model.R:8]: "the probability of transmitting infection from humans to mosquitoes with each infectious bite"?

File: R/model.R [hunk 10]
```diff
@@ -1,15 +1,54 @@
...
+#' natural_deaths: number of humans who die from aging
+#' S_count: number of humans who are Susceptible
+#' A_count: number of humans who are Asymptomatic
+#' D_count: number of humans who have the clinical malaria
+#' U_count: number of subpatent infections in humans
+#' Tr_count: number of infections being treated in humans
+#' ica_mean: the mean acquired immunity to clinical infection over the population of humans
+#' icm_mean: the mean maternal immunity to clinical infection over the population of humans
+#' ib_mean: the mean blood immunity to all infection over the population of humans
+#' id_mean: the mean immunity from detection over the population of humans
```
Comment (htopazian) [inline R/model.R:28]: detection through microscopy?

File: R/model.R [hunk 11]
```diff
@@ -1,15 +1,54 @@
...
+#' clin_inc: clinical incidence for humans between an inclusive age range (in timesteps)
+#' sev_inc: severe incidence for humans between an inclusive age range (in timesteps)
+#' E_count: number of mosquitoes in the early larval stage
+#' L_count: number of mosquitoes in the late larval stage
+#' P_count: number of mosquitoes in the pupal stage
+#' Sm_count: number of adult female mosquitoes who are Susceptible
+#' Pm_count: number of adult female mosquitoes who are incubating
+#' Im_count: number of adult female mosquitoes who are infectious
+#' total_M: number of adult female mosquitoes
+#' rate_D_A: rate that humans who transition from clinical disease to
```
Comment (htopazian) [inline R/model.R:40]: remove "who" and ditto next 2 lines

**Review by htopazian — CHANGES_REQUESTED** [review]
Nice Giovanni, I think this is very useful! Esp. as malariasimulation is launched :)

File: R/model.R (hunk 8 above)
Comment (giovannic) [reply R/model.R:31]: I've added an explainer inline

File: README.md (hunk 7 above)
Comment (giovannic) [reply README.md:22]: The line below explains that the remotes method requires RBuildTools. Is there something else I can add here?

File: R/model.R (hunk 9 above)
Comment (giovannic) [reply R/model.R:8]: hmm, it's not really a probability. It's a model specific value

File: R/model.R [hunk 12]
```diff
@@ -25,23 +27,37 @@
 #' ica_mean: the mean acquired immunity to clinical infection over the population of humans
 #' icm_mean: the mean maternal immunity to clinical infection over the population of humans
 #' ib_mean: the mean blood immunity to all infection over the population of humans
-#' id_mean: the mean immunity from detection over the population of humans
-#' pv: prevalence for humans between an inclusive age range (in timesteps)
-#' inc: incidence for humans between an inclusive age range (in timesteps)
+#' id_mean: the mean immunity from detection through microscopy over the population of humans
+#' pv: prevalence for humans between an inclusive age range (in timesteps). This
+#' defaults to pv_730_3650. Other prevalence columns can be set with
+#' prevalence_rendering_mix_ages and prevalence_rendering_max_ages parameters.
```
Comment (htopazian) [inline R/model.R:33]: mix -> min and ditto for others below

Comment (htopazian) [reply R/model.R]: fixed

File: README.md (hunk 7 above)
Comment (htopazian) [reply README.md:22]: Oh - I installed it github using remotes, but I never ran the options lines 22-24 to enable the universe. Wondering if that is needed or not

**Review by htopazian — COMMENTED** [review]
responded to a couple comments

File: README.md (hunk 7 above)
Comment (giovannic) [reply README.md:22]: the instructions for the remotes are below. They don't require universe. Perhaps it's easier to see on the [rendered README](https://github.com/mrc-ide/malariasimulation/tree/doc/outputs#installation)?

**Review by htopazian — APPROVED** [review]
Added total_M_# and an "also" to the readme file. 

Looks good to go!

**Pull request merged**